package.json

```diff
@@ -18,7 +18,8 @@
     "test": "NODE_PATH=\"./javascript/src:./admin/javascript/src\" jest",
     "coverage": "NODE_PATH=\"./javascript/src:./admin/javascript/src\" jest --coverage",
     "thirdparty": "gulp thirdparty",
-    "lint": "eslint gulpfile.js & eslint javascript/src & eslint admin/javascript/src"
+    "lint": "eslint gulpfile.js & eslint javascript/src & eslint admin/javascript/src",
+    "add-git-hooks": "./node_modules/.bin/git-hooks --install; git config merge.ours.driver true"
```

Comment:
You should be able to just do `git-hooks --install; git config merge.ours.driver true`. See https://docs.npmjs.com/misc/scripts#path